CIP-????/README.md

```diff
@@ -0,0 +1,85 @@
+--
+CIP: ?
+Title: Explicit script return values
+Category: Plutus
+Status: Proposed
+Authors:
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Implementors: 
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2024-01-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Today, Plutus Core scripts signal success or failure exclusively by whether the script terminates normally or abnormally (with an error).
+This leads to some false positives, where a script terminates normally, but this was not intended by the author.
+We propose to additionally look at what the script evaluates to when checking for success or failure.
+
+## Motivation: why is this CIP necessary?
+
+Consider the following Plutus scripts, intended to be used as minting policy scripts:
+
+```
+\redeemer -> \datum -> \script-context -> (builtin error)
+```
+
+```
+\redeemer -> \script-context -> (con false)
+```
```

Comment:
Is there a GitHub language type that causes this Plutus syntax not to be marked as errors?

Reply:
Somehow GitHub changed its parsing in the last 4 or 5 months; not marked as errors today. 🤔 (resolving)